lang/lang_en_es.txt

```diff
@@ -1236,8 +1236,200 @@
 
 #
 "Y-correct:"
-"Y-correcion:"
+"Correccion-Y:"
 
 #MSG_OFF
 " [off]"
+" [apagado]"
+
+#
+"Back"
+"atras"
+
+#
+"Checks"
+"Comprobaciones"
+
+#
+"False triggering"
+"Falsa activacion"
+
+#
+"FINDA:"
+"\x00"
+
+#
+"Firmware   [none]"
+"Firmware[ninguno]"
+
+#
+"Firmware [strict]"
+"Firmware[estrict]"
+
+#
+"Firmware   [warn]"
+"Firmware  [aviso]"
+
+#
+"HW Setup"
+"Configuracion HW"
+
+#
+"IR:"
+"\x00"
+
+#
+"Magnets comp.[N/A]"
+"Comp. imanes [N/A]"
+
+#
+"Magnets comp.[Off]"
+"Comp. imanes [Off]"
+
+#
+"Magnets comp. [On]"
+"Comp. imanes  [On]"
+
+#
+"Mesh         [3x3]"
+"Malla        [3x3]"
+
+#
+"Mesh         [7x7]"
+"Malla        [7x7]"
+
+#
+"Mesh bed leveling"
+"Nivelacion Malla Base"
+
+#
+"MK3S firmware detected on MK3 printer"
+"Firmware MK3S detectado en impresora MK3"
+
+#
+"MMU Mode [Normal]"
+"Modo MMU [Normal]"
+
+#
+"MMU Mode[Stealth]"
+"Modo MMU[Silenci]"
+
+#
+"Mode change in progress ..."
+"Cambio de modo progresando ..."
+
+#
+"Model      [none]"
+"Modelo  [ninguno]"
+
+#
+"Model    [strict]"
+"Modelo [estricto]"
+
+#
+"Model      [warn]"
+"Modelo    [aviso]"
+
+#
+"Nozzle d.  [0.25]"
+"Diam. nozzl[0.25]"
+
+#
+"Nozzle d.  [0.40]"
+"Diam. nozzl[0.40]"
+
+#
+"Nozzle d.  [0.60]"
+"Diam. nozzl[0.60]"
+
+#
+"Nozzle     [none]"
+"Nozzle  [ninguno]"
+
+#
+"Nozzle   [strict]"
+"Nozzle [estricto]"
+
+#
+"Nozzle     [warn]"
+"Nozzle    [aviso]"
+
+#
+"G-code sliced for a different level. Continue?"
+"Codigo G laminado para un nivel diferente. ?Continuar?"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"Codigo G laminado para un nivel diferente. Por favor relamina el modelo de nuevo. Impresion cancelada."
+
+#
+"G-code sliced for a different printer type. Continue?"
+"Codigo G laminado para un tipo de impresora diferente. ?Continuar?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"Codigo G laminado para una impresora diferente. Por favor relamina el modelo de nuevo. Impresion cancelada."
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"Codigo G laminado para nuevo firmware. ?Continuar?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"Codigo G laminado para nuevo firmware. Por favor actualiza el firmware. Impresion cancelada."
+
+#
+"PINDA:"
 "\x00"
+
+#
+"Preheating to cut"
+"Precalentando para laminar"
+
+#
+"Preheating to eject"
+"Precalentar para expulsar"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Diametro nozzle impresora difiere de cod.G. ?Continuar?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Diametro nozzle Impresora difiere de cod.G. Comprueba los valores en ajustes. Impresion cancelada."
+
+#
+"Rename"
+"Renombrar"
+
+#
+"Select"
+"Seleccionar"
+
+#
+"Sensor info"
+"Info sensor"
+
+#
+"Sheet"
+"Lamina"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]